api/TimeAddressableMediaStore.yaml

```diff
@@ -1624,18 +1678,47 @@ paths:
           schema:
             default: false
             type: boolean
+        - name: verbose_storage
+          in: query
+          description: |
+            Include storage metadata in `get_urls`.
+            When `verbose_storage` is `false` only `url`, `presigned`, and `label` will be included in `get_urls`.
+          schema:
+            default: false
+            type: boolean
         - name: accept_get_urls
```

Comment:
Is it implied that when all query parameters are supplied they behave as an AND? Just thinking whether it is worthwhile being specific about what happens if accept_get_urls, accept_storage_ids and presigned are all specified by the client at the same time?